scaling-controller.go

```diff
@@ -127,7 +127,7 @@ func (c *ScheduledScalerController) scheduledScalerHpaCronAdd(scheduledScaler *s
 		return
 	}
 	hpaClient := c.kubeClient.AutoscalingV1().HorizontalPodAutoscalers(scheduledScaler.Namespace)
-	hpa, err := hpaClient.Get(scheduledScaler.Spec.Target.Name, metav1.GetOptions{})
+	hpa, err := hpaClient.Get(context.TODO(), scheduledScaler.Spec.Target.Name, metav1.GetOptions{})
```

Comment:
this line causes a panic.

```
--- FAIL: TestScheduledScalerController_scheduledScalerHpaCronAdd (0.00s)
    --- FAIL: TestScheduledScalerController_scheduledScalerHpaCronAdd/empty_SS (0.00s)
panic: reflect: Call using string as type v1.GetOptions [recovered]
	panic: reflect: Call using string as type v1.GetOptions [recovered]
	panic: reflect: Call using string as type v1.GetOptions```

I have not had much success getting to the root of the issue though.

Reply:
Confirming same test failure locally. Looking into it.